### PR TITLE
add basic timing to answerQuestion

### PR DESCRIPTION
**Pull request by snopoke**
https://dimagi-dev.atlassian.net/browse/USH-826

This adds very basic timings to `answerQuestion` requests via Sentry breadcrumbs.

To get any more granularity will require one of the following:
1. adding sentry to the core library
2. moving code out of core
3. adding a layer of abstraction to allow timing in core without needing Sentry directly

(3) is likely the only real option

### CODE REVIEW

File: src/main/java/org/commcare/formplayer/application/FormController.java [hunk 1]
```diff
@@ -130,19 +127,44 @@ public FormEntryResponseBean changeLocale(@RequestBody ChangeLocaleRequestBean c
...
+                Constants.TimingCategories.PROCESS_ANSWER,
+                () -> formEntrySession.answerQuestionToJSON(
+                        answerQuestionBean.getAnswer(), answerQuestionBean.getFormIndex()
+                )
+        );
+
+        FormplayerSentry.timedBreadcrumb(
+                Constants.TimingCategories.GET_SESSION,
+                () -> formSessionService.getSessionById(answerQuestionBean.getSessionId())
+        );
```
Comment (gherceg) [inline src/main/java/org/commcare/formplayer/application/FormController.java:155]: Is this duplicate timing necessary for `GET_SESSION` considering the same call is timed a few lines earlier?

**Review by gherceg — APPROVED** [review]
One non-blocking comment. Looks good.

File: src/main/java/org/commcare/formplayer/application/FormController.java (hunk 1 above)
Comment (snopoke) [reply src/main/java/org/commcare/formplayer/application/FormController.java:155]: hmm, no, not sure why that's there.

Comment (snopoke) [inline src/main/java/org/commcare/formplayer/application/FormController.java:155]:
```suggestion
```

**Pull request merged**